ogc/src/main/scala/geotrellis/server/ogc/conf/Conf.scala

```diff
@@ -30,4 +37,27 @@ object Conf {
 
   lazy val conf: Conf = pureconfig.loadConfigOrThrow[Conf]
   implicit def ConfObjectToClass(obj: Conf.type): Conf = conf
+
+  implicit def nameConfigReader: ConfigReader[opengis.wms.Name] =
```

Comment:
I think it is slightly more idiomatic to piggyback off of previously defined readers a la:
```scala  implicit val expressionReader: ConfigReader[Expression] =
    ConfigReader[String].map { expressionString =>
      decode[Expression](expressionString) match {
        case Right(success) =>
          success
        case Left(err) =>
          throw err
      }
    }```